Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:boron
+
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+COPY package.json /usr/src/app/
+RUN npm install -q
+
+COPY . /usr/src/app
+
+RUN npm run installation -- -d
```

Comment:
It probably should be. Although, it does need the `-d` flag so it should be: 
```
RUN npm run config -- -f -d
```
The `-d` flag tells the install script that this is running in Docker so it knows that it is in a headless environment and it uses environment variables for config values. 
The `-f` flag overwrites any `config.json` file already there.